__tests__/data/indicative_output.json

```diff
@@ -0,0 +1,187 @@
+[
+  {
+    "anonymousId": "c82cbdff-e5be-4009-ac78-cdeea09ab4b1",
+    "context": {
+      "device": {
+        "id": "df16bffa-5c3d-4fbb-9bce-3bab098129a7R",
+        "manufacturer": "Xiaomi",
+        "model": "Redmi 6",
+        "name": "xiaomi"
+      },
+      "network": {
+        "carrier": "Banglalink"
+      },
+      "os": {
+        "name": "android",
+        "version": "8.1.0"
+      },
+      "traits": {
+        "address": {
+          "city": "Dhaka",
+          "country": "Bangladesh"
+        },
+        "anonymousId": "c82cbdff-e5be-4009-ac78-cdeea09ab4b1"
+      }
+    },
+    "event": "spin_result",
+    "integrations": {
```

Comment:
will this will be part of output?

Reply:
No. Output file is not updated. Doing that.